### DISCUSSION TITLE
YouTube channel

### DISCUSSION THREAD

**Discussion by ghost** [tauri-apps/tauri#1776] [General]
Hi everyone!
You have an official Youtube channel, but it is currently abandoned. 
At the moment, Tauri has a complex and confusing documentation. 
Many developers have to not only learn a new technology, but also cross the language barrier, since not everyone speaks English well. 
Videos with simple examples would be a very good solution!

**Comment by nothingismagick** [comment]
We know, but have decided to focus first on the code, then the documentation. Thanks
